tests/src/sanity.rs

```diff
@@ -521,7 +520,6 @@ mod tests {
         .await;
         let res = process.child.wait().await.unwrap();
         assert!(res.success());
```

Comment:
we want to assert the std err, but just let it be more permissive (allow specific text)

Reply:
should it be a separate function? what kind of abstraction are you looking for to handle this? it makes sense to keep it a separate function because its specific to python only

Reply:
yup, feel free to make it a standalone function or w.e you think